src/dk/aau/cs/ds306e18/tournament/rlbot/RLBotPack.java

```diff
@@ -0,0 +1,20 @@
+package dk.aau.cs.ds306e18.tournament.rlbot;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class RLBotPack {
+
+    /**
+     * @return The folder of bots called the RLBotPack. It is downloaded with the RLBotGUI. It is not guaranteed to
+     * exist and can be null if APPDATA is not an environment variable.
+     */
+    public static Path getPathToRLBotPack() {
+        try {
+            return Paths.get(System.getenv("APPDATA")).getParent().resolve("Local\\RLBotGUI\\RLBotPackDeletable");
```

Comment:
I'm fairly sure this method of getting 'appdata' stuff is Windows-centric. Maybe we should look into making a portable solution or user-accessible workaround if not found.

Reply:
The RLBot framework only works on Windows currently. Let's look into that when Linux support is relevant.

Reply:
Yeah, that's fine by me.